source/02_installation/sources-pip.rst

```diff
@@ -41,7 +41,7 @@ Follow these steps:
 
 * Install with python (sudo needed)::
 
-    sudo pip install .
+    sudo pip install . -e
```

Comment:
Why use -e, it's for development only

Reply:
Sure ... but this part of the doc is *This procedure is useful only if you want to be able to hack the code base.* ... so I suppose that you need the develop mode, no? :wink:

Reply:
Oh yes, not saw it... so ok